packages/sdk/contractkit/src/kit.ts

```diff
@@ -66,6 +67,7 @@ export interface NetworkConfig {
   validators: ValidatorsConfig
   downtimeSlasher: DowntimeSlasherConfig
   blockchainParameters: BlockchainParametersConfig
+  grandaMento: GrandaMentoConfig
```

Comment:
This breaks if the ck/cli is used with a network that doesn't have Granda Mento yet. Thi is actually true for every other contract (other than stabletoken and exchange that are handled in another way). Is this a problem or we want to keep backward compatibility with networks?

Reply:
https://github.com/celo-org/celo-monorepo/pull/8537